dali/kernels/tensor_view.h

```diff
@@ -368,6 +368,45 @@ struct TensorListView : TensorListViewBase<Backend, DataType, sample_ndim> {
   }
 };
 
+struct StorageCPU;
+struct StorageGPU;
+
+/// @brief Wraps raw memory as a tensor
```

Comment:
Maybe name them make_X (similarly to make_tuple etc).

Reply:
Fair enough. I think I can suffer 2 more characters in the identifier.